### PR TITLE
Fix some typos and verbal problem

### PR DESCRIPTION
**Pull request by nomandhoni-cs**
- :hammer: Fixed a typos in the documentation
- :hammer: Fixed a verbal problem

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -299,7 +299,7 @@ contribute.
     [our Carbon release post on GitHub](https://github.com/carbon-language/carbon-lang/discussions/1020)
     and [star carbon-lang](https://github.com/carbon-language/carbon-lang).
 -   To join the design discussion, join our
-    [our GitHub forum](https://github.com/carbon-language/carbon-lang/discussions).
+    [GitHub forum](https://github.com/carbon-language/carbon-lang/discussions).
```
Comment (jonmeow) [inline README.md:302]: FYI, I believe this was separately fixed in another PR if you merge from trunk.

File: explorer/ast/README.md [hunk 2]
```diff
@@ -36,7 +36,7 @@ representing individual syntactic constructs, such as `If` for if-statements.
 
 Sometimes it is useful to work with a subset of node types that "cuts across"
 the primary class hierarchy. Rather than deal with the pitfalls of multiple
-inheritance, we handle these cases using a form of type erasure: we specify a
+inheritances, we handle these cases using a form of type erasure: we specify a
```
Comment (jonmeow) [inline explorer/ast/README.md]:
```suggestion
inheritance, we handle these cases using a form of type erasure: we specify a
```

This is a term: https://en.wikipedia.org/wiki/Multiple_inheritance

File: README.md [hunk 3]
```diff
@@ -236,7 +236,7 @@ and with a smooth evolutionary path.
 Safety, and especially
 [memory safety](https://en.wikipedia.org/wiki/Memory_safety), remain key
 challenges for C++ and something a successor language needs to address. Our
-initial priority and focus is on immediately addressing important, low-hanging
+initial priority and focus are on immediately addressing important, low-hanging
```
Comment (jonmeow) [inline README.md]:
```suggestion
initial priority and focus is on immediately addressing important, low-hanging
```

`is` seems more correct here "priority and focus" is referring to the same thing. Addressing ow-hanging fruit is our priority and focus.

**Review by jonmeow — CHANGES_REQUESTED** [review]
Thanks! I've added some notes where I think we intend a different reading.

File: README.md (hunk 1 above)
Comment (nomandhoni-cs) [reply README.md:302]: @jonmeow I forked from the trunk today.

Can you help me solve 'pre-commit' problem in windows.

Comment (nomandhoni-cs) [reply README.md:302]: @jonmeow I have updated and formatted. Used your Suggested Commit. Please have a look

Comment (nomandhoni-cs) [reply README.md:302]: @jonmeow This feel kind of odd. When some one made pull same pull request after me and get merged.
[gh pr checkout 1631](https://github.com/carbon-language/carbon-lang/pull/1631)

Comment (chandlerc) [reply README.md:302]: Sorry for any confusion, just a bunch of different folks helping spot and fix the same typos and bugs at the same time. =D We still totally appreciate the contributions and help.

**Pull request closed without merging**